### ISSUE TITLE
'next steps' should omit `git init` if we are already in a git repo

### ISSUE THREAD

**Issue by Rich-Harris** [sveltejs/cli#287]
If you're creating a new app inside a git repo, 'next steps' includes this...

```bash
git init && git add -A && git commit -m "Initial commit" (optional)
```

...and it should probably just say nothing

**Comment by dominikg** [comment]
i'd be in favor of turning the option around entirely
automatically call `git init` unless --no-git-init was passed or we are in a git project already.
if one was initialized, add "we initialized git for you, if you don't need that, run `rm -rf .git" to next steps

**Comment by AdrianGonz97** [comment]
I think we should just get rid of it. It's unnecessary clutter for people that already know git, and it's a line full of funny words and symbols for those that don't. It's not very helpful in either case

**Comment by BlankParticle** [comment]
I think best option would be to ask to initialise a git repo if git is installed and the directory is already not a git repo.

example from `create-remix`
<img width="560" alt="image" src="https://github.com/user-attachments/assets/4dc4f568-9885-48bd-9878-f5bc066c5ec3">

**Comment by benmccann** [comment]
> i would assume that nowadays 95% are using git

I think that's about right

> once we integrate the other tools more deeply

Which tools were you thinking of?

**Comment by manuel3108** [comment]
> > i would assume that nowadays 95% are using git
> 
> I think that's about right

So why bother about the other vcs? As we have done a number of times for other decision, we shouldn't bother about those minor minorities.



> > once we integrate the other tools more deeply
> 
> Which tools were you thinking of?

I was especially thinking about `migrate` for whatever reason. Background is that currently i'm at max using `sv` 11 times (10 adders + 1 project creation) and I don't think it's worth the effort and trouble at this point.

But as soon as we implement #278 i.e. you might be using `sv` an unlimited amount of time, where it might make sense to add such prefernces.

**Comment by benmccann** [comment]
Yeah, fair enough to not ask about the other version control software. Maybe we can combine the two questions then into a single question:
```
Would you like a git repo to be created?
- yes, with a commit for each integration
- yes, with a single initial commit
- no
```

**Comment by AdrianGonz97** [comment]
I'm still against adding a git option entirely, but _if I had to choose_, simply initializing the repo like this: https://github.com/sveltejs/cli/issues/287#issuecomment-2493853391 should be sufficient. 

> Would you like a git repo to be created?
> - yes, with a commit for each integration
> - yes, with a single initial commit
> - no

I don't think a `commit per add-on` option provides any real value

**Comment by jycouet** [comment]
The current state: 

<img width="645" height="236" alt="Image" src="https://github.com/user-attachments/assets/ea611e3d-24ab-4f04-8e44-a5594f0603dd" />

---

Reading comments, and would say
```ts
function should_we_display_this_line() {
  if (we_are_in_a_git_repo) return ❌
  if (it_s_an_optional_thing) return ❌
  if (people_do_this_via_ide) return ❌
  if (people_prefers_the_message "This is where it all begins") return ❌
  if (we_dont_want_new_questions) return ❌
  if (people_are_here_they_have_npm_they_probably_know_git) return ❌
  
  // code not reachable
  return ✅
}
```

What are the good reason to keep it ?
- Making aware that git exist for new people. _But then, they need to install it and it's probably deviating from `sv` focus._
- Remind people that saving your initial state is a good idea.

---

Why not just removing this git optional thing ? :)
Looks light

<img width="402" height="220" alt="Image" src="https://github.com/user-attachments/assets/7f13fe0d-0c7f-488c-9aee-67c53d07e25b" />

**Comment by benmccann** [comment]
I'm fine removing it
